data/json/monsters/feral_humans.json

```diff
@@ -58,6 +58,55 @@
       "DROPS_AMMO"
     ]
   },
+  {
+    "id": "mon_feral_sapien_spear",
+    "type": "MONSTER",
+    "name": { "str": "feral sapien" },
+    "description": "In this feral's eyes is more intelligence than in most of their kind.  They appear to be holding a spear.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 80,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "@",
+    "color": "magenta",
+    "aggression": 30,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 4,
+    "melee_dice_sides": 6,
+    "melee_cut": 2,
+    "dodge": 3,
+    "armor_bash": 7,
+    "armor_cut": 7,
+    "armor_bullet": 6,
+    "special_attacks": [
+      {
+        "id": "stretch_attack",
+        "cooldown": 10,
```

Comment:
Did you mean to have this with a cooldown of 5?

Reply:
Yes, wanted to make it 5. It's still a bit slower than the player when using a spear but it's not too slow.

Reply:
```suggestion
        "cooldown": 5,
```

Reply:
Already made it 5.